spec/global_merge_backends_spec.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+RSpec.describe Global do
+  subject(:global) { described_class }
+
+  describe 'merging backends' do
+    # rubocop:disable RSpec/VerifiedDoubles
```

Comment:
this can be fixed and written correctly

Reply:
in this particular case doubles are fine. global takes an object that quacks like a configuration. this spec tests merging, thus abstract doubles are legit.